### PR TITLE
Added guidance on SSL cert errors

### PR DESCRIPTION
**Pull request by rmbielby**
## Overview of changes

I've added a paragraph on what analysts should do if they're blocked by an SSL cert type error.

### CODE REVIEW

File: learning-development/r.qmd [hunk 1]
```diff
@@ -592,6 +592,21 @@ Then restart R-Studio and downloads with renv should now succeed.
 
 ---
 
+### Packages not downloading (or not able to connect to other external service from within R, Python, Bash etc.) due to SSL certificate error
```
Comment (cjrace) [inline learning-development/r.qmd:595]: Is there a more concise title you could give this? Thinking the nav is already getting a bit out of hand as it is for section headers being on multiple lines
image

Maybe something like 'SSL connection errors'?

File: learning-development/r.qmd [hunk 2]
```diff
@@ -592,6 +592,21 @@ Then restart R-Studio and downloads with renv should now succeed.
...
+
+---
+
+The DfE firewall currently blocks access to external sites from DfE laptops when 
+connecting from common analytical software such as R or Python. This can prevent
+analysts from accessing important external resources such as package 
+repositories and data API endpoints. If you get an error including text along 
+the lines of "self signed SSL certificate not allowed" or anything similar 
+involving SSL certs., then contact the [Statistics Development Team ](mailto:statistics.development@education.gov.uk) with a copy of the command
```
Comment (cjrace) [inline learning-development/r.qmd:604]: spell out certificate rather than cert.? Unless that's how they're referenced in the error messages?

Comment (cjrace) [inline learning-development/r.qmd:604]: stray whitespace in the team email link text

**Review by cjrace — CHANGES_REQUESTED** [review]

File: learning-development/r.qmd (hunk 1 above)
Comment (rmbielby) [reply learning-development/r.qmd:595]: Fair point... I'll shorten it!

File: learning-development/r.qmd (hunk 2 above)
Comment (rmbielby) [reply learning-development/r.qmd]: Done, think it might have been certs in some of the error messages, but have put both options in  now

Comment (rmbielby) [reply learning-development/r.qmd]: Done

**Review by cjrace — APPROVED** [review]

**Pull request merged**